frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinitionsForm.tsx

```diff
@@ -42,18 +42,12 @@ const buildConfig = (name, networkType, typeParamsData): NetworkAttachmentDefini
   }
 
   if (networkType === 'cnv-bridge') {
-    // cnv-bridge should not define type on root
-    delete config.type;
-
-    config.plugins = [
-      {
-        type: 'cnv-bridge',
-        bridge: _.get(typeParamsData, 'bridge.value', ''),
-        vlan: parseInt(typeParamsData?.vlanTagNum?.value, 10) || undefined,
-        ipam,
-      },
-      { type: 'cnv-tuning' },
-    ];
+    Object.assign(config, {
+      bridge: _.get(typeParamsData, 'bridge.value', ''),
+      vlan: parseInt(typeParamsData?.vlanTagNum?.value, 10) || undefined,
+      macspoofchk: _.get(typeParamsData, 'macspoofchk.value', false),
```

Comment:
@metalice I talked with PM this morning and we think it should be enabled by default. That way it also follows the default advertised on docs and what is the default on RHV. Would you mind changing it? Sorry about the late notice

Reply:
sure, now its true by default